JsonPatch/JsonPatch.cs

```diff
@@ -116,10 +115,6 @@ public override JsonPatch Read(ref Utf8JsonReader reader, Type typeToConvert, Js
 	/// <param name="options">An object that specifies serialization options to use.</param>
 	public override void Write(Utf8JsonWriter writer, JsonPatch value, JsonSerializerOptions options)
 	{
-#if NET6_0_OR_GREATER
-		JsonSerializer.Serialize(writer, value.Operations, JsonPatchSerializerContext.Default.IReadOnlyListPatchOperation);
-#else
-		JsonSerializer.Serialize(writer, value.Operations);
-#endif
+		JsonSerializer.Serialize(writer, value.Operations, options);
```

Comment:
I don't understand.  Why are we not including the JTI now?

Reply:
I'm putting the Serialize methods back the way they were. JsonSerializer.Serialize will find the JTIs through the options class.